sdk/servicebus/Microsoft.Azure.Management.ServiceBus/src/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure ServiceBus management functions for managing the Microsoft Azure ServiceBus service.")]
 
 [assembly: AssemblyVersion("2.0.0.0")]
```

Comment:
You need to bump the `AsemblyVersion` if it's just a major upgrade. See the [guide here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

> AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed